packages/utils/package.json

```diff
@@ -26,6 +26,7 @@
     "ajv": "^8.6.2",
     "ajv-formats": "^2.1.1",
     "axios": "^0.21.1",
+    "bignumber.js": "^9.0.1",
```

Comment:
IIRC this is a different `BigNumber` library than [ethers uses](https://github.com/ethers-io/ethers.js/blob/b1458989761c11bf626591706aa4ce98dae2d6a9/package.json#L95), with slightly different capabilities (this lib can handle decimals, while the `BN` lib cannot), but the same interface. Mixing these two classes has historically been problematic with types, and it would be ideal if they could all use the same library

Reply:
If the decimals functionality is needed for handling approximations, we can leave it but everything leaving these functions should be parsed to an integer string

Reply:
yeah, BN is definitely different from BigNumber of ethers. but ethers BigNumber has a disadvantage when processing float numbers. so we discussed this with @rhlsthrm before using it. I think we need to definitely use bignumber.js. so the results would be more correct.

Reply:
Yeah, makes sense. One of the biggest annoyances.

We should just make sure that functions that take in this type from the interface aren't exported, and that the results are converted to integer strings when they return from the exported functions